### ISSUE TITLE
Unit test in ExchangerTest keeps failing on CI

### ISSUE THREAD

**Issue by cyrille-artho** [javapathfinder/jpf-core#294]
On Travis CI (and also my local machine), the following unit test keeps failing now:

```
 gov.nasa.jpf.test.java.concurrent.ExchangerTest > testTimeoutExchange FAILED
    java.lang.AssertionError at ExchangerTest.java:59
```

The test is from 2017. Perhaps a recent code or platform change is responsible for this test now failing? Checking this would be useful so we can either fix or disable this test.

**Comment by mmuesly** [comment]
This effects #296 and the jitpack.io distribution as well. I will have try to find the root cause in the next few days.

**Comment by mmuesly** [comment]
I can reproduce locally that 57c4cae2e builds perfectly and starting with f33b4aa2 the CI is broken. This must be a side-effect of the changes made in #295 and is not related to the build system. I am going to investigate it further.

**Comment by cyrille-artho** [comment]
Thanks, there may indeed be a problem with #295. CI was not running back then so we did not see the issue.
The problem is *not* a test dependency issue; I can run only the ExchangerTest in isolation, and it still fails:
```
./gradlew test --tests=ExchangerTest
```
So I don't know what the problem is. The code in #295 seems to only add functionality, not modify existing functionality, so it is very surprising that existing code breaks from it... but perhaps I've missed something.

**Comment by cyrille-artho** [comment]
Closed by PR #297.
